modules/cms/classes/Page.php

```diff
@@ -88,8 +89,24 @@ public function getLayoutOptions()
         $layouts = Layout::listInTheme($theme, true);
         $result = [];
         $result[null] = Lang::get('cms::lang.page.no_layout');
+        $ignoreNames = FileDefinitions::get('ignoreFiles');
+
         foreach ($layouts as $layout) {
             $baseName = $layout->getBaseFileName();
+
```

Comment:
The problem is that it just looks for the exact file name and not for patterns (=> dosn't resolve problems with filtering folders). However ignore patterns doesn't exists for [October\Rain\Filesystem\Definitions](https://github.com/octoberrain/filesystem/blob/master/Definitions.php#L45).
Idea 1.: Adding ignore file patterns to filesystem definitions and using it
Idea 2.: Check folder names of files and filter every folder with ignore files
What do you prefer?